ansible_ai_connect/ai/api/tests/test_views.py

```diff
@@ -2416,6 +2422,34 @@ def test_ok(self):
         self.assertEqual(r.data["format"], "markdown")
         self.assertEqual(r.data["explanationId"], explanation_id)
 
+    def test_ok_with_model_id(self):
+        explanation_id = str(uuid.uuid4())
+        model = "mymodel"
+        payload = {
+            "content": """---
+    - name: Setup nginx
+      hosts: all
+      become: true
+      tasks:
+        - name: Install nginx on RHEL9
+          ansible.builtin.dnf:
+            name: nginx
+            state: present
+    """,
+            "explanationId": explanation_id,
+            "ansibleExtensionVersion": "24.4.0",
+            "model": model,
+        }
+        self.client.force_authenticate(user=self.user)
+        with self.assertLogs(logger="root", level="DEBUG") as log:
```

Comment:
Should we not be asserting somewhere that `mymodel` was used!?!

Reply:
Yes. We should be able to check the `modelName` value in segment events....